src/rust-crypto/libolm_migration.ts

```diff
@@ -83,6 +83,19 @@ export async function migrateFromLegacyCrypto(args: {
     }
 
     await legacyStore.startup();
+
+    let account;
```

Comment:
type please

Reply:
Well spotted, thanks. Fixed in 8b85c6955